api/src/migrations/1-user.js

```diff
@@ -0,0 +1,48 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('users', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      name: {
+        type: Sequelize.STRING
+      },
+      email: {
+        type: Sequelize.TEXT
+      },
+      password: {
+        type: Sequelize.TEXT
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      },
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      },
+      thirty_days_from: {
+        allowNull: true,
+        type: Sequelize.DATE
+      },
+      ballotTitle: {
+        allowNull: true,
+        type: Sequelize.STRING
+      },
+      ballotDescription: {
+        allowNull: true,
+        type: Sequelize.TEXT
+      },
+      meetUpAddress: {
+        allowNull: true,
+        type: Sequelize.STRING
+      }
+    });
+  },
+  down: (queryInterface, Sequelize) => {
+    return queryInterface.dropTable('users');
+  }
+}
```

Comment:
migration to setup the user table, tested with seeders

Reply:
meetUpAddress is a great addition

Reply:
looks great!